server/src/domain/system-config/system-config.service.ts

```diff
@@ -1,6 +1,8 @@
-import { Inject, Injectable } from '@nestjs/common';
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import axios from 'axios';
```

Comment:
We should not be importing `axios` directly inside of `domain/` this should be replaced with a repository and the implementation can exist in `infra/`